apps/common/lib/image.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Common.Image do
```

Comment:
imo the structs like image, mod, modfile and tag should be in a submodule, like `Common.Struct.*` (and the files in a corresponding folder)

Reply:
we can do that. I don't really care, we won't get that much structs.

Reply:
i just like to keep code relatively organized is all. like should be organized with like and all that